frontend/src/utils/server-utils.ts

```diff
@@ -616,7 +622,16 @@ export function formatFeatureInfo(
   labelMap?: { [key: string]: string },
 ): string {
   if (type === DataType.Date) {
-    return `${moment(value).utc().format('MMMM Do YYYY, h:mm:ss')} UTC`;
+    return new Date(value).toLocaleString('default', {
```

Comment:
Why is this not using `getFormattedDate`? Would it make sense to use this code in that function instead of instantiating all those strings?